packages/README.md

```diff
@@ -0,0 +1,84 @@
+# Rx Bloc CLI
```

Comment:
Do we need the rx_bloc_cli readme file in this directory? This was probably moved here by mistake.

Reply:
Deleted. Probably  I have ran 'cp README.md ../' in wrong directory.